### PR TITLE
Conformance test for the inclusive gateway

### PR DESCRIPTION
**Pull request by CTI777**
Related to https://github.com/Activiti/Activiti/issues/2785

### CODE REVIEW

File: activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    private SecurityUtil securityUtil;
+
+    @Autowired
+    private ProcessAdminRuntime processAdminRuntime;
+
+    @Before
+    public void cleanUp() {
+        clearEvents();
+    }
+
```
Comment (eromano) [inline activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java]: double space

File: activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                        VariableEvent.VariableEvents.VARIABLE_CREATED,
+                        ProcessRuntimeEvent.ProcessEvents.PROCESS_STARTED,
+                        BPMNActivityEvent.ActivityEvents.ACTIVITY_STARTED,
+                        BPMNActivityEvent.ActivityEvents.ACTIVITY_COMPLETED,
+                        BPMNSequenceFlowTakenEvent.SequenceFlowEvents.SEQUENCE_FLOW_TAKEN,
+                        BPMNActivityEvent.ActivityEvents.ACTIVITY_STARTED,
+                        VariableEvent.VariableEvents.VARIABLE_CREATED,
+                        TaskRuntimeEvent.TaskEvents.TASK_CREATED,
+                        TaskRuntimeEvent.TaskEvents.TASK_ASSIGNED);
+
```
Comment (eromano) [inline activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java]: double space

File: activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+        tasks = taskRuntime.tasks(Pageable.of(0, 50));
+        assertThat(tasks.getTotalItems()).isEqualTo(1);
+        
+        assertThat(tasks.getContent())
+        .extracting(Task::getStatus, Task::getName)
+        .contains(
+                tuple(  task2.getStatus(),
+                        task2.getName())
+        );
+
```
Comment (eromano) [inline activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java]: double space

File: activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+                    ),
+            tuple(  BPMNActivityEvent.ActivityEvents.ACTIVITY_COMPLETED,
+                    "endEvent",
+                    "theEnd"
+                    ));
+        
+        clearEvents();
+
+    }
+
```
Comment (eromano) [inline activiti-spring-conformance-tests/activiti-spring-conformance-set4/src/test/java/org/activiti/spring/conformance/set4/BasicInclusiveGatewayTest.java]: double space

**Comment by CTI777** [comment]
Fixed

**Pull request merged**